homeassistant/components/velux/__init__.py

```diff
@@ -2,31 +2,35 @@
 import logging
 
 from pyvlx import PyVLX, PyVLXException
-import voluptuous as vol
 
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP
-from homeassistant.helpers import discovery
-import homeassistant.helpers.config_validation as cv
+from homeassistant.core import HomeAssistant
 
 DOMAIN = "velux"
 DATA_VELUX = "data_velux"
 PLATFORMS = ["cover", "scene"]
 _LOGGER = logging.getLogger(__name__)
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
I don't think you can remove the config_schema when you still allow YAML config / import.

Reply:
I did test the importing of the existing configuration for configuration.yaml to config entry multiple times on my dev machine and it did import fine. But maybe I am overlooking something.

Reply:
The CONFIG_SCHEMA should still used to validate the YAML, thus removing it will remove this validation step.

Reply:
Makes sense. Thanks, will re-add this.

Reply:
added again